cli.py

```diff
@@ -0,0 +1,236 @@
+import argparse
+import ghlib
+import jiralib
+import os
+import sys
+import json
+import util
+
+
+def fail(msg):
+    print(msg)
+    sys.exit(1)
+
+
+def serve(args):
```

Comment:
Looks like this is missing an implementation...

Reply:
Agreed, I hope to find some time today to polish the CLI a bit more.